src/main/java/edu/utdallas/davisbase/representation/CreateTableCommandRepresentation.java

```diff
@@ -0,0 +1,37 @@
+package edu.utdallas.davisbase.representation;
+
+import net.sf.jsqlparser.statement.create.table.ColumnDefinition;
+
+import java.util.List;
+
+public class CreateTableCommandRepresentation implements CommandRepresentation {
+
+    private String command;
+    private String table;
+    private List<ColumnDefinition> definitions;
+
+  public CreateTableCommandRepresentation(String command, String table, List<ColumnDefinition> definitions) {
+    this.command = command;
+    this.table = table;
+    this.definitions = definitions;
```

Comment:
```suggestion
    this.definitions = Collections.unmodifiableList(new ArrayList<>(definitions));
```

(for compile-time defensive programming and run-time fail-fast deep immutability)